docs/src/modules/components/InterfaceApiPage.tsx

```diff
@@ -131,26 +186,17 @@ export default function ApiPage(props) {
           language="jsx"
         />
 
-        {componentDescription ? (
```

Comment:
Wondering what `componentDescription` was used for - is it okay to discard completely?

Reply:
Yes, component descript does not exist for interfaces. It has `interfaceDescription` which is already used a bit above in 

```jsx
<Typography
  variant="h5"
  component="p"
  className="description"
  gutterBottom
  dangerouslySetInnerHTML={{ __html: interfaceDescription }}
/>
```

It's visible for example in [GridActionsColDef API](https://mui.com/x/api/data-grid/grid-actions-col-def/)

![image](https://github.com/user-attachments/assets/642cd804-1f58-4f60-8e38-56d96263f27c)


I replace the usual API page description 

> API reference docs for the React {{componentName}} component. Learn about the props, CSS, and other APIs of this exported module.

But that could probably be improved. I will check with other X memebers what they want for this page